### DISCUSSION TITLE
Account Switcher

### DISCUSSION THREAD

**Discussion by mirronake** [RareDevs/Rare#295] [Ideas]
I 've been looking for an Epic Games launcher for a very long time ( alternative I mean ) so that it would be possible to switch between accounts there. It is very annoying to log out and log in.

**Comment by loathingKernel** [comment]
It can't be done easily with the current infrastructure. For example, even logout-login has its issues as multiple accounts on the same configuration isn't something legendary supports or provisions for. We have been thinking about supporting multiple accounts, but Rare itself will have to be restarted every time.
